testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
@:isVar eliminates the need for a separate m_Foo for physical storage.

without @:isVar , it looks like this:

public var foo(get, set): Int;
private var m_Foo : Int;
private function get_foo() : Int {
   return m_Foo;
}
private function set_foo(x : Int) : void {
   m_Foo = x;
}


with @:isVar, it is as follows:

@:isVar public var foo(get, set): Int;
private function get_foo() : Int {
   return this.foo;
}
private function set_foo(x : Int) : void {
   this.foo = x;
}

Here, foo is backed up with a physical copy - so, no need for m_Foo.
That physical copy cannot be directly accessed though... because, 'foo' is declared as property.
Access to the property 'foo' is still governed by property's get/set declaration.

@:isVar public var foo(get, set) : Int;

is different from

public var foo : Int;

Hope that makes sense.